content/node/trustlessStaking.md

```diff
@@ -0,0 +1,120 @@
+
+**Before proceeding, please consult with your preffered Node Operator if they support this method of staking.**
+
+This document will show how to stake a node with 15k tokens and later upstake it to 60k.
+We stake with 15k first to minimize any loss resulting from mistake(eg. Typing incorrect output address or public key) 
+
+Non-Custodial Staking method involves the following addresses/wallets
+
+**Output Address**
+Never share the private key of your output address. Make ample backups of this!
+You should already have this, a wallet with your funds. This can also be referred to as the non-custodial address.
+
+
+**Operator Address**
+An Operator Address will be created by the Node Operator.  Node operator will manage(edit Domain/Chains) and run your node using the key of the operator address. 
+
+The very first stake initiated from the output address can establish a link between the operator address and the output address. Node Operator can then start running your node using the operator address. Any node rewards will be sent to the output address. Additionally; if unstaked, the unstaked funds will also go to the output address. An output address can be linked to multiple operator addresses
+
+## Introduction
+
+**Staking Bins**:
+You can stake 15k,30k,45k,60k tokens. A 30k pocket node will earn the same as two 15k nodes for instance. It is a good practice to stake 10 tokens on top of that.
+See [stake-weighted-servicer-rewards](/learn/economics/nodes/#stake-weighted-servicer-rewards) for more info
+
+
+
+**How non-custodial looks like on Poktscan**
+
+https://poktscan.com/node/bab0bc034cbf4ae82a5069b677e8c2f39098e361
+![](/images/poktscan-noncustodial.png)
+
+Note the stake type “Non-Custodial”. Also note that the address of the node is different from the output address. In the example above:
+
+**Operator Address:**  bab0bc034cbf4ae82a5069b677e8c2f39098e361  
+**Operator Address Public Key:** d7a3f683341a5fd82adf4f4411544527ba15726d9558c5b83b9a2c6946b4df44  
+**Output Address:** 3574ca25de8744aa167cff234b69b8a4eef97711
+
+Notice there is a balance of 36.57 present. Node requires some pokt to pay to claim rewards and also any staking operations(edit domain/chain) node operator will do. Either you or the node operator can keep the operator account topped up periodically.
+
+**IMPORTANT:**  Any balance in the operator address is accessible by the node operator, do not send any amount other than what you intend to be used as fees or better let your node operator handle all fees.
+
+***Checklist***
+- [ ] My output address is _________________. I want my staked funds(when unstaked) and rewards here. I have backed it up safely and I will not share the privatekeys/keyfile of this address ever.
```

Comment:
> and rewards here.

and rewards to be sent here.

> privatekeys/keyfile

private keys/keyfile

Reply:
> My output address is _________________.

Not sure exactly how to word it but we should remove this line and update to something like:

I know my output address and I want my staked funds (when unstaked) and rewards to be sent here.